src/test-function-specifier-config.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * A declarative way to target a function call either imported from a package,
+ * like node:test or from another file in the project
+ */
+export interface TestFunctionSpecifierConfig {
+  /** The names of the functions that should be included in the test runner view */
+  name: string[] | string;
```

Comment:
Maybe this should always be an array - I didn't realize the JSON schema in the package.json was going to be awkward to express this union type.